e2e/test/scenarios/visualizations-tabular/pivot_tables.cy.spec.js

```diff
@@ -1376,7 +1376,7 @@ describe("scenarios > visualizations > pivot tables", { tags: "@slow" }, () => {
     cy.findByTestId("viz-type-button").click();
     cy.findByTestId("Pivot Table-button").click();
     cy.wait("@pivotDataset");
-    cy.findAllByTestId("pivot-table-cell")
+    cy.findByTestId("pivot-table")
```

Comment:
@uladzimirdev found that this is more performant.
Please see https://github.com/metabase/metabase/pull/45066/files#r1665508219